src/filtrering/lagrede-filter/lagrede-filter_innhold.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {Radio} from 'nav-frontend-skjema'
+import RedigerKnapp from '../../components/knapper/rediger-knapp';
+import {
+    avmarkerLagretFilter,
+    LagretFilter_ActionReducers,
+    velgLagretFilter,
+} from '../../ducks/lagret-filter_action-reducers';
+import {AppState} from '../../reducer';
+import {FiltervalgModell} from '../../model-interfaces';
+import {lagredeFilterListerErLik} from "../../components/modal/lagrede-filter/lagrede-filter-utils";
+import {finnSideNavn} from "../../middleware/metrics-middleware";
+import {logEvent} from "../../utils/frontend-logger";
+import {LagreFilterModal, Visningstype} from "../../components/modal/lagrede-filter/lagre-filter-modal";
+import './lagrede-filter_innhold.less'
+
+interface LagredeFilterInnholdProps {
+    lagretFilter: LagretFilter_ActionReducers[]
+    filterValg?: FiltervalgModell;
+    filtergruppe?: string;
+}
+
+function LagredeFilterInnhold(props: LagredeFilterInnholdProps) {
+    const outerDivRef = useRef<HTMLDivElement>(null);
+    const className = (props.lagretFilter.length >= 18) ? 'lagrede-filter__valgfelt__lang' : 'lagrede-filter__valgfelt'
+
+    const [visEndreFilterModal, setVisEndreFilterModal] = useState(false);
+    const filtreringMinOversikt = useSelector((state: AppState) => state.filtreringMinoversikt);
+
+    const dispatch = useDispatch();
+
+    const valgtFilter = props.lagretFilter.find(elem => lagredeFilterListerErLik(elem.filterValg, filtreringMinOversikt));
+    const finnLagretFilter = (filterId) => props.lagretFilter.find((elem) => elem.filterId === parseInt(filterId));
+
+    const velgFilter = (filterId: string) => {
+        const filterVerdi = finnLagretFilter(filterId);
+
+        logEvent('portefolje.metrikker.lagredefilter.velg-gruppe',
+            {}, {filterId: filterVerdi!.filterId, sideNavn: finnSideNavn()});
+        dispatch(velgLagretFilter(filterVerdi!));
+    };
+
+    useEffect(() => {
+        valgtFilter
+            ? velgFilter(valgtFilter.filterId.toString())
+            : dispatch(avmarkerLagretFilter());
+    }, [valgtFilter, dispatch]);
```

Comment:
@alexandraback hjelp her, får feilmelding om at "velgFilter" skal være i dependency, men må bruke useCallback hvis den er det etcetcetc.... får ikke buildet pga denne

Reply:
gör som felmeldingen siger ;)

Reply:
Fixed, thanks to @alexandraback